config/clusters/2i2c/ohw.values.yaml

```diff
@@ -1,7 +1,16 @@
 dask-gateway:
   gateway:
     nodeSelector:
-      2i2c.org/community: ohw
+      k8s.dask.org/node-purpose: core
```

Comment:
Are 2i2c providing such labels on the nodes? Otherwise, just not setting a nodeSelector will be fine - it won't schedule on user dedicated nodes as they have a taint making it not schedule on those anyhow.

I could guess 2i2c provided `hub.jupyter.org/node-purpose: core` labels, because the jupyterhub helm chart relates to that out of the box - but the dask-gateway helm chart does not relate to `k8s.dask.org/node-purpose` label nor does its documentation recommend configuring that etc.